lib/resolveLang.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var defaultLangCode = require('./defaultLangCode'),
+	normalizeLangCode = require('./normalizeLangCode');
+
+var locales = [
```

Comment:
Yep think you are right.  So many [specs and revisions](http://en.wikipedia.org/wiki/IETF_language_tag).  It remarks about how language codes have been used as locale identifiers in many applications - that's probably where my confusion came form (i.e. probably Microsoft or IBM).  Anyway, it looks like  [RFC 5646] (https://www.ietf.org/rfc/rfc5646.txt) is more recent... if we make reference in readme or anywhere, we might want to refer to newer version? - It looks like a lot of the revisions were to normalize some of the formatting from various specs around the sub-keys.